apps/agreement/contracts/Agreement.sol

```diff
@@ -93,18 +93,20 @@ contract Agreement is ILockManager, IAgreement, AragonApp {
     }
 
     struct Challenge {
-        uint256 actionId;                   // Identification number of the action associated to the challenge
-        address challenger;                 // Address that challenged the action
-        uint64 endDate;                     // Last date the submitter can raise a dispute against the challenge
-        bytes context;                      // Link to a human-readable text providing context for the challenge
-        uint256 settlementOffer;            // Amount of collateral tokens the challenger would accept without involving the arbitrator
-        uint256 arbitratorFeeAmount;        // Amount of arbitration fees paid by the challenger in advance
-        ERC20 arbitratorFeeToken;           // ERC20 token used for the arbitration fees paid by the challenger in advance
-        ChallengeState state;               // Current state of the action challenge
-        bool submitterFinishedEvidence;     // Whether the action submitter has finished submitting evidence for a raised dispute
-        bool challengerFinishedEvidence;    // Whether the action challenger has finished submitting evidence for a raised dispute
-        uint256 disputeId;                  // Identification number of the dispute on the arbitrator
-        uint256 ruling;                     // Ruling given for the action's dispute
+        uint256 actionId;                      // Identification number of the action associated to the challenge
+        address challenger;                    // Address that challenged the action
+        uint64 endDate;                        // Last date the submitter can raise a dispute against the challenge
+        bytes context;                         // Link to a human-readable text providing context for the challenge
+        uint256 settlementOffer;               // Amount of collateral tokens the challenger would accept without involving the arbitrator
+        uint256 challengerArbitratorFeeAmount; // Amount of arbitration fees paid by the challenger in advance
+        ERC20 challengerArbitratorFeeToken;    // ERC20 token used for the arbitration fees paid by the challenger in advance
+        uint256 submitterArbitratorFeeAmount;  // Amount of arbitration fees paid by the challenger in advance
+        ERC20 submitterArbitratorFeeToken;     // ERC20 token used for the arbitration fees paid by the challenger in advance
```

Comment:
What do you think about having a separate object like:
```js
struct ArbitratorFees { 
   ERC20 token;
   uint256 amount;
}
```

so we can do sth like:
```js
struct Challenge {
   ...
   ArbitratorFees submitterArbitratorFees;
   ArbitratorFees challengerArbitratorFees;
   ...
}
```

Reply:
This way we use one extra SSTORE per Challenge. Do you still prefer it this way?

Reply:
Yea